README.md

```diff
@@ -57,3 +57,11 @@ WIDTH=4 RADIUS=8 CROSSWALK=True CROSSWALK_LENGTH=2 empy3 templates/intersection.
 ## Resources
 
 Example files with different values are provided under the `resources` folder. 
+
+## Standalone maps
+
+Standalone maps that doesn't respond to a particular template are added in the `standalone` folder.
+
+ - [12_map_integration](standalone/12_map_integration/) map: A large-combined map was created out of the available `resources`. See [12_map_integration/README](standalone/12_map_integration/README.md) for more information about the map creation.
```

Comment:
Should add "process" after "creation".

Should also wrap this line at 120 characters.

Reply:
Done